### PR TITLE
Added spells to text pc template 

### PR DESCRIPTION
**Pull request by ZakariyaRguibi**
a proposed fix for #195

### CODE REVIEW

File: Library/Output Templates/Text - PC.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ Disadvantages:@DISADVANTAGES_LOOP_START @DESCRIPTION_PRIMARY [@POINTS];@DISADVAN
 Quirks:@QUIRKS_LOOP_START @DESCRIPTION_PRIMARY [@POINTS];@QUIRKS_LOOP_END
 Skills:@SKILLS_LOOP_START @DESCRIPTION_PRIMARY @RSL [@POINTS]-@SL;@SKILLS_LOOP_END
 Equipment:@EQUIPMENT_LOOP_START @QTY @DESCRIPTION_PRIMARY [$@COST; @WEIGHT];@EQUIPMENT_LOOP_END
+Spell:@SPELLS_LOOP_START @DESCRIPTION_PRIMARY [Class: @CLASS; Cost: @MANA_CAST; Maintain: @MANA_MAINTAIN; Time: @TIME_CAST ; Duration:@DURATION];@SPELLS_LOOP_END
```
Comment (richardwilkes) [inline Library/Output Templates/Text - PC.txt:13]: Shouldn't this be "Spells:", i.e. plural, like all the rest?

Comment (richardwilkes) [reply Library/Output Templates/Text - PC.txt:13]: Also, unlike all the other things that cost points, you're not listing the points spent in the spell, which seems wrong.

Comment (ZakariyaRguibi) [reply Library/Output Templates/Text - PC.txt]: done, added RSL as well

File: Library/Output Templates/Text - PC.txt [hunk 2]
```diff
@@ -10,6 +10,7 @@ Disadvantages:@DISADVANTAGES_LOOP_START @DESCRIPTION_PRIMARY [@POINTS];@DISADVAN
 Quirks:@QUIRKS_LOOP_START @DESCRIPTION_PRIMARY [@POINTS];@QUIRKS_LOOP_END
 Skills:@SKILLS_LOOP_START @DESCRIPTION_PRIMARY @RSL [@POINTS]-@SL;@SKILLS_LOOP_END
 Equipment:@EQUIPMENT_LOOP_START @QTY @DESCRIPTION_PRIMARY [$@COST; @WEIGHT];@EQUIPMENT_LOOP_END
+Spells:@SPELLS_LOOP_START @DESCRIPTION_PRIMARY @RSL [@POINTS] [Class: @CLASS; Cost: @MANA_CAST; Maintain: @MANA_MAINTAIN; Time: @TIME_CAST ; Duration:@DURATION];@SPELLS_LOOP_END
```
Comment (richardwilkes) [inline Library/Output Templates/Text - PC.txt:13]: And one more: @SL should probably be there, for the actual skill level, which is almost always the thing you want. @RSL is rarely needed for spells, but could be useful.

Comment (ZakariyaRguibi) [reply Library/Output Templates/Text - PC.txt]: done !

**Review by richardwilkes — APPROVED** [review]

**Pull request merged**